src/ScottPlot5/ScottPlot/Control/Backend.cs

```diff
@@ -0,0 +1,187 @@
+﻿using ScottPlot.Control.EventArgs;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace ScottPlot.Control
+{
+    public class Backend
+    {
+        private MouseDownEventArgs? CurrentMouse1Down = null;
+        private MouseDownEventArgs? CurrentMouse2Down = null;
+        private MouseDownEventArgs? CurrentMouse3Down = null;
+
+        private readonly HashSet<Key> CurrentlyPressedKeys = new();
+        private Pixel? LastMousePosition = null;
+
+        private readonly Plot plot;
+        private readonly object EventSender;
+        private readonly Action requestRender;
```

Comment:
I'm a little unclear what this is to be used for. It's private but not called. Is it here so the backend can eventually request its own renders (e.g., after a delay)?

Reply:
It's here because the events handlers can request renders, but since the event handlers are set up in the constructor we don't actually need to save this in a field.

Reply:
I think this is resolved now using generic event handlers